libraries/botbuilder-dialogs-adaptive/src/recognizers/crossTrainedRecognizerSet.ts

```diff
@@ -63,8 +70,15 @@ export class CrossTrainedRecognizerSet extends Recognizer implements CrossTraine
 
         const results = await Promise.all(
             this.recognizers.map(
-                (recognizer: Recognizer): Promise<RecognizerResult> => {
-                    return recognizer.recognize(dialogContext, activity, telemetryProperties, telemetryMetrics);
+                async (recognizer: Recognizer): Promise<RecognizerResult> => {
```

Comment:
You should be able to omit the explicit types in this function as they are inferred by the Typescript compiler.